### ISSUE TITLE
Why does ModelCheckpoint not monitor `val_loss`?

### ISSUE THREAD

**Issue by kvantricht** [tensorflow/similarity#282]
Hi guys,

I was going through the `unsupervised_hello_world.ipynb` notebook and noticed [here](https://github.com/tensorflow/similarity/blob/067abafc0f27a491635501c4476193a7bfd4849e/examples/unsupervised_hello_world.ipynb?short_path=93b7732#L988) that the `ModelCheckpoint` callback is monitoring the training loss. Given the fact that there's also validation data being used in the training, what is the reason that it's not `val_loss` which is monitored to identify the best model?

**Comment by owenvallis** [comment]
Hi @kvantricht, thanks for catching this. I just pushed an update in [45b047d](https://github.com/tensorflow/similarity/commit/45b047dd2f224c40f9abe7df9aaaf4346410af04)
